README.md

```diff
@@ -266,6 +267,7 @@ For those cases, additional properties (apart from `message`) are included:
     * `withCredentials` [`<Boolean>`][] - Passed to the request library to make CORS requests. **Default:** `false`
     * `payloadStructure` [`<String>`][] - (*LogDNA usage only*) Ability to specify a different payload structure for ingestion. **Default:** `default`
     * `compress` [`<Boolean>`][] - (*LogDNA usage only*) Compression support for the agent. **Default:** `false`
+    * `httpProxy` [`<String>`][] - The full URL of an http or https proxy to pass through
```

Comment:
@darinspivey, can we change the variable name to `proxy` here?

Reply:
Sure thing

Reply:
done!